gemma/tokenizer.cc

```diff
@@ -114,57 +114,96 @@ bool GemmaTokenizer::Decode(const std::vector<int>& ids,
   return impl_->Decode(ids, detokenized);
 }
 
+GemmaChatTemplate::GemmaChatTemplate(const GemmaTokenizer& tokenizer) {
+  Init(tokenizer);
```

Comment:
Let's inline this? Just looking at the header, it's not clear whether Init should be called by users.